index.js

```diff
@@ -127,15 +137,19 @@ export function persistentMap(prefix, initial = {}, opts = {}) {
   }
 
   onMount(store, () => {
-    let data = { ...initial }
-    for (let key in storageEngine) {
-      if (key.startsWith(prefix)) {
-        data[key.slice(prefix.length)] = decode(storageEngine[key])
+    let sync = () => {
```

Comment:
Let’s move `sync` out of `onMount` (to `persistentMap` level). We don’t need to redefine it every mount.

Reply:
Done